nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestBulkManagementObject.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Nour AL KOTOB
+ */
+package org.nuxeo.rest.management;
+
+import static javax.servlet.http.HttpServletResponse.SC_NOT_FOUND;
+import static org.junit.Assert.assertEquals;
+import static org.nuxeo.rest.management.TestRenditionsObject.PICTURES_RECOMPUTE_PATH;
+
+import java.io.IOException;
+
+import org.junit.Test;
+import org.nuxeo.jaxrs.test.CloseableClientResponse;
+import org.nuxeo.runtime.test.runner.Deploy;
+
+/**
+ * @since 11.1
+ */
+@Deploy("org.nuxeo.ecm.platform.picture.core")
+public class TestBulkManagementObject extends ManagementBAFTest {
+
+    @Test
+    public void testGetStatus() throws IOException {
+        runBulkAction(PICTURES_RECOMPUTE_PATH, null);
+    }
+
+    @Test
+    public void testGetStatusWithWrongCommandId() {
+        try (CloseableClientResponse response = httpClientRule.get("site/management/bulk/fakeCommandId")) {
```

Comment:
Could concatenate the suggested new constant from `ManagementBAFTest` with the `fakeCommandId`.